### PR TITLE
Add git metadata to the package to assist in troubleshooting.

### PR DESCRIPTION
**Pull request by dwighthubbard**

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: redislite/__init__.py [hunk 1]
```diff
@@ -27,12 +27,20 @@
     'package_metadata.json'
 )
 
-if os.path.exists(_metadata_file):  # pragma: no cover
+if os.path.exists(_metadata_file):
     with open(_metadata_file) as fh:
         _package_metadata = json.load(fh)
```
Comment (zosimos) [inline redislite/__init__.py]: Just wondering, is there any possibility that package_metadat.json might have invalid json format?

**Comment by dwighthubbard** [comment]
It's generated during the run of setup.py so if it has invalid json the package is broken.

**Comment by zosimos** [comment]
:thumbsup: LGTM

**Pull request merged**